packages/shared/src/graphql/squads.ts

```diff
@@ -294,6 +300,17 @@ export const joinSquadInvitation = (
   params: SquadInvitationProps,
 ): Promise<SquadInvitation> =>
   request(`${apiUrl}/graphql`, SQUAD_JOIN_MUTATION, params);
+
+export const checkIsHandleTaken = async (handle: string): Promise<boolean> => {
```

Comment:
Do we use this taken tense?
Not quite sure we have other examples wether we prefer positive or negative tense.

(eg: handleAvailable vs handleTaken) or isExistingHandle) 
not blocking, but if we have something similar let's keep it in line.

Reply:
Since we have use-cases for `exists` already, let's keep it to that. Will refactor this once the API is ready for merging. Thanks for noting! 🚢